plugin/src/types.ts

```diff
@@ -35,8 +36,12 @@ export type WorkletizableFunction =
 export const WorkletizableFunction =
   'FunctionDeclaration|FunctionExpression|ArrowFunctionExpression|ObjectMethod';
 
+export type WorkletizableObject = ObjectExpression;
+
+export const WorkletizableObject = 'ObjectExpression';
+
 export function isWorkletizableFunctionType(
-  path: NodePath
+  path: NodePath<unknown>
```

Comment:
Why was this needed?

Reply:
By default NodePath has its generic parameter as Node, which doesn't allow nulls etc.